### ISSUE TITLE
Bug: Typing letters or portions of filenames in a Files window causes unpredictable scrolling behavior

### ISSUE THREAD

**Issue by JollyRoger8X** [files-community/Files#14119]
### Description

In a Files window with lots of files with names ranging from A-W, if I type the letter "a", the very first filename starting with the letter "a" is selected and the window is scrolled to that position as expected.

However, if I then type the letter "i", the first filename starting with the letter "i" is selected but the window scrolls to the very *top* of the list alphabetically and *does not* scroll to the position of the selected file. 

Likewise, if I type the letter "p", the first filename with the letter "p" is selected but the window scrolls to a seemingly random position *below* filenames starting with that letter.

### Steps To Reproduce

1. Open a FIles window to a directory with many files in it with names starting with letters ranging the alphabet.
2. Type a letter and observe the scrolling behavior.
3. Type some more letters and observe the scrolling behavior.

### Requirements

As a user, I expect File to select the first file whose name starts with the letter(s) I type, and I expect Files to scroll to show the selected file.

### Files Version

Preview 3.0.13.0

### Windows Version

10.0.19045.3324

### Log File

No relevant logs

**Comment by yaira2** [comment]
> As a user, I expect File to select the first file whose name starts with the letter(s) I type, and I expect Files to scroll to show the selected file.

This is how it already works, can you share a screen recording if you're seeing otherwise?

**Comment by JollyRoger8X** [comment]
The recording file is too large to upload to Github which limits the file size to 10 MB.

You should be able to access the link I gave you though. It works fine for me in Edge and Chrome.
Are you seeing an error message?

**Comment by yaira2** [comment]
I still can't repro this, please let me know if I'm following the correct steps
1. Open `C:\Windows\System32`
2. Type "note" to select "notepad.exe"
3. Type "appv" to select the "AppV" folder
4. Type "wri" to select "write.exe"

**Comment by JollyRoger8X** [comment]
Nope., it is not working correctly. 

Here's a shorter video showing what happens in the `System32` directory:

https://github.com/files-community/Files/assets/6462046/161fc7a4-839a-455d-8eb3-b968e1cb4fd8

**Comment by JollyRoger8X** [comment]
Unfortunately I do not have Windows 11 running on any of my machines at the moment. I'll follow up if this changes.

**Comment by yaira2** [comment]
Thanks! We'll keep tracking this of course, but as of now that's the only thing I can think of.

**Comment by Josh65-2201** [comment]
Can you see if this is still an issue in the current preview version. https://files.community/download

**Comment by yaira2** [comment]
I don't think this has been resolved. From what I understand it's related to virtualization.
